server/sessionstore.go

```diff
@@ -218,5 +218,17 @@ func NewSessionStore(lifetime time.Duration) *SessionStore {
 	statsRegisterInt("LiveSessions")
 	statsRegisterInt("TotalSessions")
 
+	statsRegisterInt("IncomingMessagesWebsockTotal")
```

Comment:
It made sense to init session counters in `sessionstore.go`, but given these new initializations, maybe they should be moved to something like `statsRegisterVars()` in `stats.go`. Or maybe move to `newHub()` in `hub.go` next to 
```
statsRegisterInt("LiveTopics")
statsRegisterInt("TotalTopics")
```

Reply:
Done.